tools/VscPreviewExtension/README.md

```diff
@@ -30,7 +30,7 @@ For further information and details about DocFX Flavored Markdown, please refere
 | `ctrl+k q` | `Open Dfm Preview to the side` | Preview to side |
 |  | `show Dfm Show` | Show Source |
 
-  ![PreviewToside](/tools/VscPreviewExtension/images/previewToSide.gif)
+  ![PreviewToside](https://github.com/dotnet/docfx/raw/master/tools/VscPreviewExtension/images/previewToSide.gif)
```

Comment:
>https://github.com/dotnet/docfx/raw/master/tools/VscPreviewExtension/images/previewToSide.gif [](start = 19, length = 93)

I feel /tools/* is better since absolute path will contain "branch(master)"